application/controllers/Api.php

```diff
@@ -1792,4 +1792,131 @@ private function validateDate($date, $format = 'Y-m-d')
         return $d && $d->format($format) === $date;
     }
     
+	/**
+     * Get one node of the organization
+     * @author Mickael ROMMME
+     */
+    public function organization_node($id) {
```

Comment:
Don't use underscore and use meaningful name, such as getEntity as Organization is composed of entities